bugbug/commit_features.py

```diff
@@ -808,34 +809,47 @@ def __call__(self, commit, **kwargs):
 
 
 class Files(object):
+    name = "files"
+
+    def __call__(self, commit, **kwargs):
+        return commit["files"]
+
+
+def _pass_through_tokenizer(doc):
+    return doc
+
+
+class FrequentFilesVectorizer(BaseEstimator, TransformerMixin):
     def __init__(self, min_freq=0.0014):
         self.min_freq = min_freq
+        self.vectorizer = CountVectorizer(
+            tokenizer=_pass_through_tokenizer, lowercase=False
+        )
 
-    def fit(self, commits):
-        self.count = defaultdict(int)
+    def fit(self, X, y=None):
+        min_freq = self.min_freq
+        total_commits = len(X)
+        file_count = defaultdict(int)
 
-        self.total_commits = 0
+        for commit_files in X:
+            for filename in commit_files:
+                file_count[filename] += 1
 
-        for commit in commits:
-            self.total_commits += 1
+        frequent_files = {
+            filename
+            for filename, count in file_count.items()
+            if count / total_commits >= min_freq
+        }
```

Comment:
I wonder if we could use CountVectorizer's min_df to avoid doing this manually

Reply:
I am surprised that I didn't think of it before, but it's actually possible. Done in b2a4a8e7bc3f5f9141c777edee8a3368a0317ad0. It produces the exact same number of features.